src/Stratis.Bitcoin.IntegrationTests/BlockStore/RetrieveFromBlockStoreSteps.cs

```diff
@@ -136,8 +136,8 @@ private void a_real_transaction()
 
         private void the_block_with_the_transaction_is_mined()
         {
-            this.blockWithTransactionId = this.node.GenerateStratisWithMiner(1).Single();
-            this.node.GenerateStratisWithMiner(1);
+            this.blockWithTransactionId = TestHelper.MineBlocks(this.node, 1).BlockHashes.Single();
```

Comment:
Can these 2 lines be replaced by
`this.blockWithTransactionId = TestHelper.MineBlocks(this.firstStratisPowApiNode, 2).BlockHashes[0];`?

Reply:
Yes it can 👍